boltdesignsystem/bolt
# Feature/ds 360 card action icon override

## Jira

https://pegadigitalit.atlassian.net/browse/DS-360

## Summary

Allows users to override action icons in card components.

## Details

When adding an action to a card component, a user can specify which icon they would like shown. If no icon is specified, default logic is applied (`right-chevron` is shown for internal links and `external-link` is shown for external links.)

## How to test

- Review files changed
- Specify one of the icons from [here](http://localhost:3000/pattern-lab/?p=viewall-components-icon) in the actions an ensure that that icon is shown in place of the `chevron-right` or `external-link` icons
- Ensure that not specifying an icon falls back on the default behavior
